### DISCUSSION TITLE
Bring hacked

### DISCUSSION THREAD

**Discussion by Bearroller9640** [community/community#139263] [Code Security]
### Select Topic Area

General

### Body

Ex friend put a spoof number in my google account last November andalware now 5 phones later he is still hacking with Trojans and tasks that are hidden he and his dark web buddies are using this site and others to download and program files to send to my phone. He's uploaded hidden child documents to my phone. The police have 10 reports from me and the FBI have 8 and nothing is being done how can I stop this? Rese
![trashed-1729473378-Screenshot_20240902-182715](https://github.com/user-attachments/assets/e6dae113-c8ad-4aad-8a6e-0470fa93520b)

ts and safe mode doesn't work Trojans are at 15 now no help from anyone. Please someone help me. I don't use this site just joined to get help

**Comment by A-S-W-I-N-S-P-A-R-K-Y** [comment]
Response #1

**Reply by Bearroller9640** [reply]
Sorry if I did something wrong I'm just looking for Answers and help

**Comment by Ponk446** [comment]
I recommend u writing this into another online portal for similar problems.

you should find out the way he's installing trojans on there first.
but:
use vpn like proton vpn with kill switch on, and vpn starting on demand.
hard/factory reset.
could u sue him for that?
if he has good knowledge with ROMs then maybe get a new phone.
else there's no way to install anything. ONLY use play store (never install an .apk).
NEVER accept "installing from unknown sources" for any application and use something like "Malwarebytes Mobile Security" from playstore.
don't hand him your phone or anything like that.
always check that developer options are DEACTIVATED (settings-> developer options, should be somewhere next to backup&reset and system updates)
I hope that helps, when there's other devices in ur network check them too.
and maybe reset the router.

**Comment by ghostinhershell** [comment]
Hi @Bearroller9640,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on issues similar to this. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Reply by Ha121246** [reply]
![Uploading IMG_1769.jpeg…]()
